Update swing-card.component.ts

fobaz opened the pull request:
Forgot why did I do this, but it helps.

fobaz committed:
Update swing-card.component.ts

Forgot why did I do this, but it helps.

peterpeterparker commented:
@fobaz this PR doesn't look ok.

In `SwingCardComponent` you did following modification:

    this.card = this.swingStack.addCard(this);

Which isn't valid. `this.swingStack.addCard` is a void method and therefore doesn't send back any value.

Would you like to improve your PR or should I close/cancel it?

fobaz committed:
Update swing-stack.component.ts

Fixed the void return.

I think it's easier to get the card this way, instead of comparing elements.

fobaz commented:
Fixed the void return.

I think it's easier to get the card this way, instead of comparing elements.
